### PR TITLE
Low high renaming

### PR DESCRIPTION
**Pull request by dvadym**
This PR renames ` low and high` fields of `AggregateParams/SumParams` classes  to `min_value/max_value`.

### CODE REVIEW

File: tests/accumulator_test.py [hunk 1]
```diff
@@ -466,73 +466,6 @@ def test_with_noise(self):
                                delta=4)
 
 
-class SumAccumulatorTest(unittest.TestCase):
```
Comment (dvadym) [inline tests/accumulator_test.py:469]: This test started failed for some unclear reasons. Accumulator are already deprecated (will be removed next week), so it's easier to remove the test :)

File: pipeline_dp/accumulator.py [hunk 2]
```diff
@@ -243,8 +243,8 @@ def delta(self):
     @property
     def mean_var_params(self):
         return dp_computations.MeanVarParams(
-            self.eps, self.delta, self.aggregate_params.low,
-            self.aggregate_params.high,
+            self.eps, self.delta, self.aggregate_params.min_value,
+            self.aggregate_params.min_value,
```
Comment (Kulankhina) [inline pipeline_dp/accumulator.py:247]: max_value

File: pipeline_dp/dp_engine.py [hunk 3]
```diff
@@ -141,15 +141,19 @@ def _check_aggregate_params(self, col, params: AggregateParams,
                 "params.max_contributions_per_partition must be set "
                 "to a positive integer")
         needs_low_high = any(metric == Metrics.SUM for metric in params.metrics)
-        if needs_low_high and (params.low is None or params.high is None):
-            raise ValueError("params.low and params.high must be set")
-        if needs_low_high and (self._not_a_proper_number(params.low) or
-                               self._not_a_proper_number(params.high)):
+        if needs_low_high and (params.min_value is None or
```
Comment (Kulankhina) [inline pipeline_dp/dp_engine.py:144]: nit: "needs_low_high" -> "needs_min_max"

File: pipeline_dp/dp_engine.py [hunk 4]
```diff
@@ -141,15 +141,19 @@ def _check_aggregate_params(self, col, params: AggregateParams,
...
+                               params.max_value is None):
             raise ValueError(
-                "params.low and params.high must be both finite numbers")
-        if needs_low_high and params.high < params.low:
+                "params.min_value and params.max_value must be set")
+        if needs_low_high and (self._not_a_proper_number(params.min_value) or
```
Comment (Kulankhina) [inline pipeline_dp/dp_engine.py:148]: nit: "needs_low_high" -> "needs_min_max"

**Review by Kulankhina — APPROVED** [review]
Looks great. Thanks!

File: pipeline_dp/accumulator.py (hunk 2 above)
Comment (dvadym) [reply pipeline_dp/accumulator.py]: Thanks!

File: pipeline_dp/dp_engine.py (hunk 3 above)
Comment (dvadym) [reply pipeline_dp/dp_engine.py]: Thanks!

**Review by dvadym — COMMENTED** [review]
Thanks for review!

**Pull request merged**